synthtool/gcp/templates/node_library/.github/bug_report.md

```diff
@@ -0,0 +1,33 @@
+---
+name: Bug report
```

Comment:
This section gets formatted into a table. Is this intentional? 
![7qvs39grae0](https://user-images.githubusercontent.com/8822365/47239930-8df78280-d39b-11e8-977c-8ae6ac46b12b.png)

Reply:
Yup!  It should look like this:
https://github.com/googleapis/google-cloud-node/issues/new/choose